src/main/java/seedu/duke/project/Project.java

```diff
@@ -52,8 +61,38 @@ public void displayProjectBacklog() {
     // Call this function every time a new sprint object is instantiated.
     // sets the start date the first time.
     public void setStartDate() {
-        if (startDate != null) {
+        if (startDate == null) {
```

Comment:
@gmit22 Can I double check with you if this is supposed to be `==` or `!=`?

Reply:
Yes that makes more sense for setting it. Thanks